weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ScorersPage/AnnotationScorerForm.tsx

```diff
@@ -0,0 +1,89 @@
+import {Box} from '@material-ui/core';
+import React, {FC, useCallback, useState} from 'react';
+import {z} from 'zod';
+
+import {createBaseObjectInstance} from '../wfReactInterface/baseObjectClassQuery';
+import {TraceServerClient} from '../wfReactInterface/traceServerClient';
+import {sanitizeObjectId} from '../wfReactInterface/traceServerDirectClient';
+import {projectIdFromParts} from '../wfReactInterface/tsDataModelHooks';
+import {ScorerFormProps} from './ScorerForms';
+import {ZSForm} from './ZodSchemaForm';
+
+const AnnotationScorerFormSchema = z.object({
+  Name: z.string().min(1),
+  Description: z.string().min(1),
+  Type: z.discriminatedUnion('type', [
+    z.object({
+      type: z.literal('boolean'),
```

Comment:
you might want to capitalize Boolean (and other properties / literals) so that they render on the schema editor as such... depends on the display layer that you want.

Reply:
I honestly think it looks fine, because its not a label, it just gets rendered in the selector dropdown.